srcpkgs/qt5/template

```diff
@@ -21,8 +20,8 @@ short_desc="Cross-platform application and UI framework (QT5)"
 maintainer="John <me@johnnynator.dev>"
 license="GPL-3.0-or-later, LGPL-3.0-or-later"
 homepage="https://qt.io/"
-distfiles="http://download.qt.io/official_releases/qt/${version%.*}/${version}/single/qt-everywhere-src-${version}.tar.xz"
-checksum=3a530d1b243b5dec00bc54937455471aaa3e56849d2593edb8ded07228202240
+distfiles="https://void.johnnynator.dev/distfiles/qt5-${version}.tar.gz"
```

Comment:
Is it feasible to document how the tarball was generated?

Reply:
Sure, done. The current one is created with slightly different ignored submodules, but I didn't want to regenerate it just to save like 5mb.

Reply:
Nice! Thanks a lot.

(Maybe we can package the archive-all thingy? Haven't looked how complex it is yet).

Reply:
Afaik it only needes Python, so it should be trivial.